src/components/molecules/TabMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import TabItem from './atoms/TabItem';
+
+interface ITabMenuProps {
+  className?: string;
+  tabLabels: Array<string>;
+}
+
+const TabMenu: React.FC<ITabMenuProps> = ({className, tabLabels, children}) => {
+  const [activeTab, setActiveTab] =  useState(0);
+
+  const handleTabClick = (index: number) => setActiveTab(index);
+
+  const renderTabs = () => (
+    tabLabels.map((e, i) => (
+      <TabItem key={e} tabIndex={i} onClick={handleTabClick} isActive={i === activeTab}>{e}</TabItem>),
+      )
+  );
+  return (
+    <React.Fragment>
+      <ul className={className} style={{listStyleType: 'none'}} >
+        {renderTabs()}
+      </ul>
+    </React.Fragment>
+
```

Comment:
This can be shortened down to

```jsx
const UL = styled.ul`
  border-bottom: 1px solid #ccc;
  padding-left: 0;
  list-style-type: none;
`;
...
  return (
    <UL>
      {renderTabs()}
    </UL>
  );
```